### PR TITLE
Workaround to support provision 4.16 rosa cluster

### PR DESCRIPTION
**Pull request by yasun1**
Workaround of the bug [OSD-21663](https://issues.redhat.com/browse/OSD-21663)

[OCPQE-21078](https://issues.redhat.com//browse/OCPQE-21078)

### CODE REVIEW

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-aws-rosa-hcp-byo-kms-oidc-auditlog-stage-full-f7

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-stable-aws-rosa-hcp-stage-critical-f14

File: ci-operator/step-registry/rosa/sts/operator-roles/delete/rosa-sts-operator-roles-delete-commands.sh [hunk 1]
```diff
@@ -22,10 +22,6 @@ ROSA_TOKEN=$(cat "${CLUSTER_PROFILE_DIR}/ocm-token")
 if [[ ! -z "${ROSA_TOKEN}" ]]; then
   echo "Logging into ${OCM_LOGIN_ENV} with offline token"
   rosa login --env "${OCM_LOGIN_ENV}" --token "${ROSA_TOKEN}"
-  if [ $? -ne 0 ]; then
```
Comment (svetsa-rh) [inline ci-operator/step-registry/rosa/sts/operator-roles/delete/rosa-sts-operator-roles-delete-commands.sh:25]: @yasun1 why are we removing this check everywhere? Any particular reason? Or is the new error message when login fails is more intuitive than this?

Comment (yasun1) [reply ci-operator/step-registry/rosa/sts/operator-roles/delete/rosa-sts-operator-roles-delete-commands.sh:25]: This is a redundant step, it is never run. If login fails, the script will be terminated at line 24. The failure message from the login is clear enough.

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-ec-aws-rosa-hcp-stage-critical-f14

File: ci-operator/step-registry/rosa/sts/operator-roles/delete/rosa-sts-operator-roles-delete-commands.sh (hunk 1 above)
Comment (svetsa-rh) [reply ci-operator/step-registry/rosa/sts/operator-roles/delete/rosa-sts-operator-roles-delete-commands.sh:25]: Response from Yanming in slack:
This is a redundant step, it is never run. If login fails, the script will be terminated at line 24. The failure message from the login is clear enough.

**Pull request merged**